transformer_engine/pytorch/csrc/extensions.cu

```diff
@@ -1084,6 +1099,11 @@ PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
     .def("split_overlap_ag", &ubuf::UbufP2PCommOverlap::split_overlap_ag)
     .def("copy_input_to_ubuf", &ubuf::UbufP2PCommOverlap::copy_input_to_ubuf)
     .def("get_ubuf_output", &ubuf::UbufP2PCommOverlap::get_ubuf_output);
+#else  // NVTE_MPI_FOUND
+  m.def("UbufOverlapAlgo", &placeholder, "Get cublasLt version");
+  m.def("UbufCommOverlap", &placeholder, "If userbuf backend is available");
+  m.def("UbufP2PCommOverlap", &placeholder, "If userbuf backend is available");
```

Comment:
These comments are incorrect.

I don't like the idea that these classes with MPI and functions without. It would be somewhat nicer if the non-MPI case were a dummy class that threw an exception in the constructor.

Reply:
These are not initialized at all when MPI is not found. The only place where they're used in that case is for the python side annotations. All we need is some dummy value so that the type checker does not throw an error. So I thought this was a simple solution without overengineering too much.

**NOTE**: On the python side we will use `tex.userbuf_comm_available()` to throw an error in case the user wants to leverage this backend without MPI being installed.

Reply:
It's fine as a quick hack, it just feels delicate to me.